udata-tabular-preview/data-structure.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="bg-alt-green-tilleul-verveine fr-p-3v fr-my-2w" v-if="hasError">
+    <p class="fr-grid-row fr-m-0">
+      <span class="fr-icon-warning-line" aria-hidden="true"></span>
+      {{ $t("The data structure of this file failed to load.") }}
+    </p>
+  </div>
+  <Loader v-if="loading" />
+  <div v-if="!hasError && !loading" class="fr-grid-row fr-grid-row--gutters">
+    <div class="fr-col-12 fr-col-sm-6 fr-col-md-4 fr-col-lg-3" v-for="column in columns">
```

Comment:
Maybe we should add a message when columns is empty (only happen for xlsx?). Example on the first ressource of [this dataset](http://dev.local:7000/en/datasets/liste-et-localisation-des-musees-de-france/).

Reply:
Do we want something like the warning on explore ?
"This is an Excel file, analysis on this type of files are limited."